site/templates/page/curated.jet

```diff
@@ -15,7 +15,7 @@
   {{ plan = page.Plans[0] }}
 {{ end }}
 
-  <main id="main" class="page curated-page {{if len(page.PageCollections) > 0}}curated-collection-page{{end}}">
+  <main id="main" class="page curated-page {{if len(page.PageCollections) > 0}}curated-collection-page{{end}} {{if plan != 0}}curated-plan-page{{end}}">
```

Comment:
Nit: I know you are following the existing style here, but can we change both to have the leading space in the `if` so that we don't end up with extra spaces when not used.

```suggestion
  <main id="main" class="page curated-page{{if len(page.PageCollections) > 0}} curated-collection-page{{end}}{{if plan != 0}} curated-plan-page{{end}}">
```